### ISSUE TITLE
missing notes/ files

### ISSUE THREAD

**Issue by dangbert** [mikonvergence/DiffusionFastForward#6]
Hi, the notes files linked in the "Notes" section of the README.md don't appear to exist (except for [01-Diffusion-Theory.md](https://github.com/mikonvergence/DiffusionFastForward/blob/master/notes/01-Diffusion-Theory.md).  Am I missing something or these files just haven't been created yet?

For example this one is missing:
https://github.com/mikonvergence/DiffusionFastForward/blob/master/notes/02-Pixel-Diffusion.md

Thanks for creating this awesome youtube series and notebooks!

**Comment by mikonvergence** [comment]
Yes, I've not had a chance to review the remaining notes due to other commitments, so they were never added to the official project.

It seems that there is some interest, so I might add this in the coming weeks!

If you have any other suggestions for the content you would like to see as part of the course, feel free to propose!

**Comment by dangbert** [comment]
Hi Mike, sounds great! If you already have a draft done of these notes and want to upload them on another branch I'd love to read over them.

Otherwise no worries, I still have to go through all the notebooks in great detail so there's a lot left for me to learn that way. Thanks again!
